examples/python/multiprocessing/server.py

```diff
@@ -92,6 +98,26 @@ def _reserve_port():
 
 
 def main():
+    # Check if we're on macOS and warn about SO_REUSEPORT limitations
+    if platform.system() == "Darwin":
+        _LOGGER.warning(
```

Comment:
I'd probably merge all this separate warning into a single `_LOGGER.warning` call with a multiline message.